Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -26,20 +26,20 @@
 
 // UserProfile properties may be read from any thread, but should
 // only be mutated when synchronized on the profile manager.
-@interface UserProfile : TSYapDatabaseObject
+@interface UserProfile : TSYapDatabaseObject <NSCopying>
```

Comment:
Don't we already get NSCopying conformance from TSYapDatabaseObject via Mantle?

Reply:
great point!